windows/devices/uwb/UwbConnector.cxx

```diff
@@ -948,31 +1134,22 @@ UwbConnector::DeregisterEventCallback(std::weak_ptr<::uwb::RegisteredCallbackTok
         // treat it as a session callback
         auto sessionId = sessionCallback->SessionId;
 
-        auto node = m_sessionEventCallbacks.extract(sessionId);
-        if (node.empty()) {
-            return; // sessionId has no associated tokens
-        }
-
-        auto tokens = node.mapped();
-        auto tokenIt = std::find_if(std::cbegin(tokens), std::cend(tokens), [sessionCallback](const auto& token) {
-            return token.get() == sessionCallback.get();
-        });
-        if (tokenIt == std::cend(tokens)) {
-            return; // no associated token found, bail
-        }
-        tokens.erase(tokenIt);
-        m_sessionEventCallbacks.insert(std::move(node));
+        DeregisterSessionEventCallback<::uwb::OnSessionEndedToken>(tokenShared, m_onSessionEndedCallbacks)                               ? 0
+            : DeregisterSessionEventCallback<::uwb::OnRangingStartedToken>(tokenShared, m_onRangingStartedCallbacks)                     ? 0
+            : DeregisterSessionEventCallback<::uwb::OnRangingStoppedToken>(tokenShared, m_onRangingStoppedCallbacks)                     ? 0
+            : DeregisterSessionEventCallback<::uwb::OnPeerPropertiesChangedToken>(tokenShared, m_onPeerPropertiesChangedCallbacks)       ? 0
+            : DeregisterSessionEventCallback<::uwb::OnSessionMembershipChangedToken>(tokenShared, m_onSessionMembershipChangedCallbacks) ? 0
+                                                                                                                                         : 0;
+        return;

```

Comment:
Is the `return` necessary here?

Reply:
+1, not needed, same with other instance below.